posthog/temporal/batch_exports/batch_exports.py

```diff
@@ -252,24 +263,23 @@ def iter_records(
 
         query_fields = ",".join(f"{field['expression']} AS {field['alias']}" for field in fields + control_fields)
 
-    lookback_days = 4
```

Comment:
I've removed this as it was deceptive: `lookback_days` is only a parameter in `SELECT_FROM_EVENTS_VIEW`, and in that view we overrode `lookback_days` from `settings.DEFAULT_TIMESTAMP_LOOKBACK_DAYS`, which is actually 7. So, having it here set to 4 was just confusing.